app/controllers/AnnotationIOController.scala

```diff
@@ -170,20 +170,19 @@ class AnnotationIOController @Inject()(val messagesApi: MessagesApi)
     }
   }
 
-  def downloadProject(projectId: String, user: User)(implicit ctx: DBAccessContext) = {
+  def downloadProject(projectId: String, user: UserSQL)(implicit ctx: DBAccessContext) = {
     for {
       projectIdValidated <- ObjectId.parse(projectId)
       project <- ProjectSQLDAO.findOne(projectIdValidated) ?~> Messages("project.notFound", projectId)
-      teamIdBson <- project._team.toBSONObjectId.toFox
-      _ <- user.assertTeamManagerOrAdminOf(teamIdBson)
+      _ <- Fox.assertBoolean(user.isTeamManagerOrAdminOf(project._team))
```

Comment:
This purely waits for the future and checks that it is not `None`, right? Seems to be the correct refactoring, still wondering what this line does.

Reply:
This is about functions that return `Fox[Boolean]`. At several locations, we want to check such booleans which we can only get packed in a Fox. One previous method to deal with that by creating helper functions for each, such as `assertTeamManagerOrAdminOf`, which unpacks the Fox returned by `isTeamManagerOrAdminOf`, and then checks the `content` of that boolean, by casting it to a new Fox via `bool2fox`. In this PR the number of these helper functions would have increased, so I refactored this into the helper function `Fox.assertBoolean`. It returns `Fox[Unit]`, which is full __only__ when the original function returns `Fox.successful(true)´.

Reply:
Ah, ok. How about renaming it to `Fox.assertTrue`?